packages/components/src/Image/src/Image.tsx

```diff
@@ -0,0 +1,82 @@
+import { type ResponsiveProp, type StyledSystemProps, useResponsiveValue, useStyledSystem } from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { type CSSProperties, type ForwardedRef, forwardRef, type HTMLProps } from "react";
+import { useContextProps } from "react-aria-components";
+
+import { type AccessibleSlotProps, type BaseComponentDOMProps, cssModule } from "../../utils/index.ts";
+
+import { ImageContext } from "./ImageContext.ts";
+
+import styles from "./Image.module.css";
+
+export const GlobalImageCssSelector = "hop-Image";
+
+export interface ImageProps extends
+    StyledSystemProps,
+    AccessibleSlotProps,
+    Omit<BaseComponentDOMProps, "children">,
+    Omit<HTMLProps<HTMLImageElement>, "slot" | "color" | "content" | "height" | "width" | "src"> {
+    /**
+     * The image shape.
+     */
+    shape?: "straight" | "rounded" | "circular";
+    /**
+     * An image path.
+     */
+    src?: ResponsiveProp<string>;
+}
+
+function Image(props: ImageProps, ref: ForwardedRef<HTMLImageElement>) {
```

Comment:
i saw this in Spectrum, and it's pretty smart. 

```

  if (!alt) {
    console.warn(
      'The `alt` prop was not provided to an image. ' +
      'Add `alt` text for screen readers, or set `alt=""` prop to indicate that the image ' +
      'is decorative or redundant with displayed text and should not be announced by screen readers.'
    );
  }
```

Reply:
what do you think?

Reply:
Yeah good idea!